pyproject.toml

```
@@ -68,6 +63,12 @@ tests = [
     "utm",
     "scikit-rf==0.31.0",
 ]
+dotnet = [
+    "cffi == 1.15.1;platform_system=='Linux' and python_version == '3.7'",
+    "cffi == 1.16.0;platform_system=='Linux' and python_version > '3.7'",
+    "pywin32 >= 303;platform_system=='Windows'",
+    "ansys-pythonnet>=3.1.0rc3",
+    "dotnetcore2 ==3.1.23;platform_system=='Linux'",]
```

```suggestion
    "dotnetcore2 ==3.1.23;platform_system=='Linux'",
]
```
Really minor change to be consistent with the rest of the targets.